lib/web/fetch/response.js

```diff
@@ -252,6 +253,13 @@ class Response {
     // clonedResponse, this’s headers’s guard, and this’s relevant Realm.
     return fromInnerResponse(clonedResponse, this[kHeaders][kGuard], this[kRealm])
   }
+
+  [nodeUtil.inspect.custom] (depth, options) {
```

Comment:
please do the same thing you did for Request here, this is missing quite a few properties (and node:util should handle the actual formatting for us, which includes new lines, adding quotes, etc.)

Reply:
thank you

Reply:
@mertcanaltin this wasn't resolved

Reply:
tried a few improvements

Reply:
The properties shouldn't rely on kState, you should copy what you did for Request: https://github.com/nodejs/undici/pull/2955/files#diff-60be498c70018c51cbe2ebcbb1952b8e91d7ca912ea1b4a39189dbb0b109651cR781-R797

Taking the public properties from the Response instance (ie. triggering the getters via this.whatever instead of this[kState].whatever), add them to an object, and inspect that object. Otherwise, having status/statusText/HeadersList isn't too useful. Here's what should be added:

  [body](https://developer.mozilla.org/en-US/docs/Web/API/Response/body)
    [bodyUsed](https://developer.mozilla.org/en-US/docs/Web/API/Response/bodyUsed)
    [headers](https://developer.mozilla.org/en-US/docs/Web/API/Response/headers)
    [ok](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok)
    [redirected](https://developer.mozilla.org/en-US/docs/Web/API/Response/redirected)
    [status](https://developer.mozilla.org/en-US/docs/Web/API/Response/status)
    [statusText](https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText)
    [type](https://developer.mozilla.org/en-US/docs/Web/API/Response/type)
    [url](https://developer.mozilla.org/en-US/docs/Web/API/Response/url)

Reply:
thank you, I have added the missing

Reply:
These should have been added to Response, not Request